purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1088,6 +1097,22 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             })
     }
 
+    private fun extractSkus(offeringsJSON: JSONObject): Set<String> {
+        val jsonArrayOfOfferings = offeringsJSON.getJSONArray("offerings")
```

Comment:
i've always wondered if it's worth pulling these json keys into constants somewhere. thoughts? probably depends on how often we use them

Reply:
What does everyone think of using [`GSON`](https://github.com/google/gson) instead?

Reply:
(By the way this method is just extracted from the existing implementation).

Reply:
we've avoided adding any 3rd party libs.... @vegaro probably has more historical info on that though, and whether we might open up our thinking..

Reply:
Oh that's right.

Reply:
We should use Kotlin Serialization https://kotlinlang.org/docs/serialization.html#example-json-serialization

I thought we had a story for it but I couldn't find it so I created one https://app.shortcut.com/revenuecat/story/12198/use-kotlin-serialization-instead-of-manual-serialization